EasyClangComplete.py

```diff
@@ -78,6 +78,21 @@ def run(self, edit):
             log.debug(" Nothing to clean")
 
 
+class ApplyFixitCommand(sublime_plugin.TextCommand):
+    """Command to apply fixit at cursor position."""
+
+    def run(self, edit):
+        """All the actual processing is done in on_text_command."""
+
+
+class ReplaceCharactersCommand(sublime_plugin.TextCommand):
+    """Command to actually replace region with fixit value."""
+
+    def run(self, edit, region, value):
+        """Replace given region with given string."""
+        self.view.replace(edit, sublime.Region(*region), value)
```

Comment:
Again, why using a new command for this? Why not just applying this line instead of this one: `view.run_command('replace_characters', fixits[index])`? It will remove the need for another class and will sumplify the workflow